packages/front-end/components/Settings/EnvironmentModal.tsx

```diff
@@ -106,8 +110,26 @@ export default function EnvironmentModal({
         placeholder=""
         textarea
       />
+      <div className="mb-3">
+        <Toggle
+          id={"defaultToggle"}
+          label="Identifier"
+          value={!!form.watch("defaultState")}
+          setValue={(value) => {
+            form.setValue("defaultState", value);
+          }}
+        />{" "}
+        <label form="defaultToggle">
+          Default state{" "}
+          <Tooltip
+            tipPosition="top"
+            text="The default state for this environment when adding new features"
+          />
```

Comment:
Instead of having a tooltip, I would just make the label say "Default state for new features"